### PR TITLE
fix(rtl): add correct text-align

### PR DESCRIPTION
**Pull request by AmitMY**
#### Short description of what this resolves:
Fixed rest of bad text alignments

**Ionic Version**: 3.x

### CODE REVIEW

**Comment by manucorporat** [comment]
@brandyscarney LGTM!

**Comment by brandyscarney** [comment]
@AmitMY @manucorporat The code itself looks great! I just want to test it real quick to make sure nothing broke. :)

File: src/themes/ionic.functions.scss [hunk 1]
```diff
@@ -291,3 +291,19 @@
 
   @return $colors-list;
 }
+
+// Sets correct text align with support for old browsers
+// @param {string} $direction - text direction
+// @param {string} $decorator - !default, !important
```
Comment (manucorporat) [inline src/themes/ionic.functions.scss]: I think !default is just a SCSS thing, right? @brandyscarney ?

Comment (brandyscarney) [inline src/themes/ionic.functions.scss]: @manucorporat Yes, TMK it is only used in Sass variable declarations.

**Comment by brandyscarney** [comment]
Snapshot comparison with master: http://ionic-snapshot-go.appspot.com/ionic2/snapshots/rrn/fw0/chrome_400x800#a2f6e0cd

Looks great! :smile:

**Pull request merged**